ext/hash/hash_xxhash.c

```diff
@@ -174,19 +174,24 @@ zend_always_inline static void _PHP_XXH3_Init(PHP_XXH3_64_CTX *ctx, HashTable *a
 			func_init_seed(&ctx->s, (XXH64_hash_t)Z_LVAL_P(_seed));
 			return;
 		} else if (_secret) {
-			if (!try_convert_to_string(_secret)) {
+			zval secret_copy;
+			ZVAL_COPY(&secret_copy, _secret);
+			if (!try_convert_to_string(&secret_copy)) {
```

Comment:
Can't this use `zval_try_get_string()` possibly instead of converting the zval?

Reply:
Maybe even use the `tmp` version?

Reply:
Sure, lemme change this quickly

Reply:
I prefer the non-temp version as it will be cleaner code, and the performance difference here will be negligible anyway.